Doc/library/exceptions.rst

```diff
@@ -90,8 +90,13 @@ The following exceptions are used mostly as base classes for other exceptions.
    .. method:: with_traceback(tb)
 
       This method sets *tb* as the new traceback for the exception and returns
-      the exception object.  It is usually used in exception handling code like
-      this::
+      the exception object.  It was more commonly used before the exception
+      chaining features of :pep:`484` became available.  The following example
```

Comment:
```suggestion
      chaining features of :pep:`3134` became available.  The following example
```

Did you meant this one?

Reply:
Yes, I think so thanks. Not sure how I came up with 484.